data/sql/updates/pending_db_world/rev_1629137505579314700.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1629137505579314700');
+
+INSERT INTO `spell_linked_spell` (`spell_trigger`, `spell_effect`, `type`, `comment`) VALUES ('17770','29940','0','Wolfshead Helm Energy');
```

Comment:
```suggestion
INSERT INTO `spell_linked_spell` (`spell_trigger`, `spell_effect`, `type`, `comment`) VALUES (17770,29940,0,'Wolfshead Helm Energy');
```

don't use string for integers

Reply:
@Helias Blame SQLyog export feature 😄